apps/mobile/src/libs/enums/app/base-color.enum.ts

```diff
@@ -3,7 +3,7 @@ const BaseColor = {
 	BG_LIGHT: "rgba(246, 245, 250, 1)",
 	BG_WHITE: "rgba(255, 255, 255, 1)",
 	BLACK: "rgba(0, 0, 0, 1)",
-	BLUE: "rgba(0, 240, 255, 1)",
+	BLUE: "rgba(64, 190, 255, 1)",
```

Comment:
Why did you change this value? It will affect every element that uses this color, is it expected?

Reply:
This color is used only by our Link component. I went to Figma and compared, that previous color was a pretty different in our design. Actually, there was a gradient color. I used a color that was close enough to what the user should see.